### PR TITLE
Update community.html

### PR DESCRIPTION
**Pull request by denshirenji**
We are adding MetricFire.com to this list of commercial support because we offer a Hosted Prometheus Service. You can find us here, at www.metricfire.com, thank you! Signed-off-by: Lindsey Rogerson <lindsey@metricfire.com>

### CODE REVIEW

File: content/community.html [hunk 1]
```diff
@@ -107,6 +107,7 @@ <h1>Commercial Support</h1>
       <li><a href="https://grafana.com">Grafana Labs</a></li>
       <li><a href="https://juliusv.com">Julius Volz</a> (independent contractor)</li>
       <li><a href="https://tina.tincho.org">Martina Ferrari</a> (independent contractor)</li>
+      <li><a href="https://metricfire.com">MetricFire</a> (SaaS)</li>
```
Comment (brian-brazil) [inline content/community.html]: SaaS shouldn't be mentioned, we don't mention what any of the others do.

**Comment by denshirenji** [comment]
Hi Brian,

Thank you! I removed "(Saas)" and added DCO, how's it looking? Does the link not work for you? It works for me but it's slow, we probably need to improve that page. Thank you!

Best,
Lindsey

**Comment by brian-brazil** [comment]
You still need to add the DCO.

Linking to https://www.metricfire.com/hosted-prometheus would be better I think.

**Comment by denshirenji** [comment]
looking good now? Thank you!

**Comment by brian-brazil** [comment]
DCO still isn't happy, try squashing.

**Pull request closed without merging**